python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -647,6 +647,29 @@ def TensorsConcatPromoteDTypeModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class CatModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([2, 2, 4], torch.float32, True),
+        ([2, 1, 4], torch.float32, True),
+        ([2, 3, 4], torch.float32, True),
+    ])
+    def forward(self, x, y, z):
+        return torch.ops.aten.cat([x, y, z], dim=1)
```

Comment:
Unless it really has a different result, please use `torch.cat` instead of `torch.ops.aten.cat`.